dataflow-website/stream-developer-guides/streams/standalone-stream-kafka/usage-detail-sender/src/main/java/io/spring/dataflow/sample/usagedetailsender/UsageDetailSender.java

```diff
@@ -1,31 +1,25 @@
 package io.spring.dataflow.sample.usagedetailsender;
 
 import java.util.Random;
+import java.util.function.Supplier;
 
 import io.spring.dataflow.sample.UsageDetail;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.cloud.stream.annotation.EnableBinding;
-import org.springframework.cloud.stream.messaging.Source;
-import org.springframework.messaging.support.MessageBuilder;
-import org.springframework.scheduling.annotation.EnableScheduling;
-import org.springframework.scheduling.annotation.Scheduled;
-
-@EnableScheduling
-@EnableBinding(Source.class)
+@Configuration
 public class UsageDetailSender {
 
-	@Autowired
-	private Source source;
-
 	private String[] users = {"user1", "user2", "user3", "user4", "user5"};
 
-	@Scheduled(fixedDelay = 1000)
```

Comment:
Without this, how are we going to repeat the same payload every 1s? Perhaps you're changing the behavior of the sample then?

Reply:
@sabbyanandan The behavior is not changed. With the new functional model of suppliers, you don't need to add the `Scheduled` annotation any more. By default, there is a poller that triggers the supplier every second. So, you will see the same behavior.

Reply:
Ah, thank you for the education.